StickKnightsOnline-Client/main.cpp

```diff
@@ -3834,6 +3868,11 @@ construct_frame()
                DrawImage(0, 244, options_popup);
 
                /* draw little options selectors */
+			   //Sign enabling
+			   if (enableSIGN)
+				   DrawImage(169, 393, option_selector);
+			   else
+				   DrawImage(206, 393, option_selector);
```

Comment:
take note of how annoying it is to type these "magic numbers" and not be able to configure them except inside main.cpp

Think to the future, how can we read these from a file? How can we make it easier to edit? Something to ponder so we can decide later :)